### PR TITLE
feat: activate saved search at it's actual url

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

File: src/router/routes.js [hunk 1]
```diff
@@ -501,13 +501,17 @@ module.exports = [
...
-		path: '/lend/saved-search-beta',
+		path: '/lend/saved-search',
 		component: () => import('@/pages/Settings/SavedSearchBeta'),
 		meta: {
 			authenticationRequired: true,
 			excludeFromStaticSitemap: true,
 		}
 	},
+	{
+		path: '/lend/saved-search-beta',
```
Comment (dyersituations) [inline src/router/routes.js:512]: Do we need this fallback?

**Review by dyersituations — APPROVED** [review]

File: src/router/routes.js (hunk 1 above)
Comment (mcstover) [reply src/router/routes.js:512]: eh prob not I guess it was only @tiarascoleman who may have memorized that one...

Comment (mcstover) [reply src/router/routes.js:512]: I'll clean it up here and also the Prod version of the puppet configs.

**Review by tiarascoleman — APPROVED** [review]
Yay!

**Pull request merged**